packages/venia-concept/src/components/clickable.css

```diff
@@ -4,7 +4,6 @@
     display: inline-flex;
     justify-content: center;
     line-height: 1;
-    padding: 0;
```

Comment:
This is not really a necessary change, I can back it out if desired.

Reply:
Clickable is essentially a mixin you would use for some shared styles between links and buttons of all kinds. You use it by writing `composes: clickable` in your component CSS. As such, it doesn't *change* the padding of a button to `0`, it *resets* the padding to `0`, like a default. Your component would override that padding as needed.